### DISCUSSION TITLE
Being able to move Companion Widget

### DISCUSSION THREAD

**Discussion by Icaruk** [dailydotdev/daily#1071] [Feature Requests]
Some webpages looks like this:

![](https://i.gyazo.com/7bd188a8a102448532adb2afcac6a0bd.png)

It could be nice to move the widget dragging, or maybe just setting height value.

**Comment by idoshamun** [comment]
Hi @Icaruk,

Thanks for the suggestion. The suggestion already exists: https://github.com/dailydotdev/daily/discussions/622
